pages/agent/v3/cli_annotate.md

```diff
@@ -38,6 +38,27 @@ steps:
 
 <%= image "annotations-styles.png", alt: "Screenshot of available annotation styles" %>
 
+## Annotation ordering
+
+Annotations are ordered by priority (highest to lowest) and then by most recently created. Priority can be set with the `--priority` option when creating the annotation with `buildkite-agent annotate`. Priority ranges from `1` to `10`, with `10` being the highest priority and `1` being the lowest. If priority is not set, the default priority is `3`. If two annotations have the same priority, the one created most recently will be displayed first.
```

Comment:
love it 🥰

Reply:
I really love communication styles like this to describe how something works. It makes it so easy to understand and naturally answers questions you have as you go along